tests/stress/taskStress_test.go

```diff
@@ -1,73 +0,0 @@
-package system
-
-import (
-	"testing"
-	"time"
-
-	"github.com/quintilesims/layer0/common/testutils"
-)
-
-// Test Resources:
-// This test creates an environment named 'tp'
-// and a deploy named 'alpine'
-func TestTaskStress(t *testing.T) {
```

Comment:
I think this test should be cleaned up a bit, but definitely not deleted, as it's the only test that creates tasks.

Reply:
Cool - I had a comment on the `232-tlake` -> `232-stress-tests` PR earlier about this; in `stress_test.go:NewStressTest()`, you had advocated for replacing the `t Tester` param with a straight-up `b *testing.B` param, which broke compatibility with these non-benchmarking tasks.

Offhand, this test doesn't make use of benchmarking, and so probably shouldn't be using `*testing.B`. Should `NewStressTest()` be modified to accept either `*testing.T` or `*testing.B`, or should `TestTaskStress()` call a new `NewXTest()` function?

Reply:
I think there's a case to be made for making tasks a param for `benchmarkStress`

Reply:
Refactored task stress test into benchmark pattern.